scenario27/resources/user.properties

```diff
@@ -41,12 +41,14 @@ UserEmail=test234@wso2.com
 
 #JDBC User Stores
 JDBCUserStoreDomainName=PRIMARY
-JDBCUserStoreUrl=jdbc:mysql://10.100.11.23:3306/is520user
+#TODO: Change the JDBC user store URL
+JDBCUserStoreUrl=jdbc:mysql://172.31.47.70:3306/is520user
```

Comment:
This will not be accessible from AWS where IAM cluster is running. So please setup this in a AWS instance and provide the url here.

Reply:
Yes, I have added a TODO (in the line above) to change the values. This commit only includes the basic support for parameterization as this scenario is currently blocked in terms of implementation